sdk/identity/identity/src/credentials/azurePowerShellCredential.ts

```diff
@@ -166,13 +166,8 @@ export class AzurePowerShellCredential implements TokenCredential {
       ]);
 
       const result = results[1];
```

Comment:
I'm really loving the approach taken by dotnet https://github.com/Azure/azure-sdk-for-net/blob/09a650ffd29ed3d531224edb4d47cd51e8aeb9b5/sdk/identity/Azure.Identity/src/Credentials/AzurePowerShellCredential.cs#L245-L263 which does seem more resilient - Should we do that here as well?

Reply:
We'll have to revisit our logic of extracting token and splitting the Get-AzAccessToken and ConvertTo-Json as two different steps for the -SecureString flag.